entry/package.json

```diff
@@ -9,7 +9,5 @@
   "description": "example description",
   "repository": {},
   "license": "ISC",
-  "dependencies": {
-    "@ohos/materialchips": "file:./materialchips"
-  }
+  "dependencies": {}
```

Comment:
The lib dependency must be present here i.e entry's package.json. Remove the dependency from project's package.json